contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -572,23 +571,19 @@ The API supports three different PATCH operations, determined by their
 corresponding Content-Type header:
 
 * JSON Patch, `Content-Type: application/json-patch+json`
- * As defined in [RFC6902](https://tools.ietf.org/html/rfc6902), a JSON Patch is
+  * As defined in [RFC6902](https://tools.ietf.org/html/rfc6902), a JSON Patch is
 a sequence of operations that are executed on the resource, e.g. `{"op": "add",
 "path": "/a/b/c", "value": [ "foo", "bar" ]}`. For more details on how to use
 JSON Patch, see the RFC.
 * Merge Patch, `Content-Type: application/merge-patch+json`
- * As defined in [RFC7386](https://tools.ietf.org/html/rfc7386), a Merge Patch
+  * As defined in [RFC7386](https://tools.ietf.org/html/rfc7386), a Merge Patch
 is essentially a partial representation of the resource. The submitted JSON is
 "merged" with the current resource to create a new one, then the new one is
 saved. For more details on how to use Merge Patch, see the RFC.
 * Strategic Merge Patch, `Content-Type: application/strategic-merge-patch+json`
- * Strategic Merge Patch is a custom implementation of Merge Patch. For a
+  * Strategic Merge Patch is a custom implementation of Merge Patch. For a
 detailed explanation of how it works and why it needed to be introduced, see
-below.
-
-#### Strategic Merge Patch
-
-Details of Strategic Merge Patch are covered [here](/contributors/devel/sig-api-machinery/strategic-merge-patch.md).
+[here](/contributors/devel/sig-api-machinery/strategic-merge-patch.md).
```

Comment:
Note to reviewers: I got rid of this section in the markdown document because the whole section was just a link to another doc describing the strategic merge patch and the bullet list entry directly above here contained virtually the exact phrase as what was contained in this section. So, I just condensed the whole section to be a link to the strategic merge patch document from the bulleted list entry above.